apps/api/src/api.module.ts

```diff
@@ -2,14 +2,14 @@ import { Module } from '@nestjs/common';
 import { EventEmitterModule } from '@nestjs/event-emitter';
 import { BullModule } from '@nestjs/bullmq';
 import { ScheduleModule } from '@nestjs/schedule';
-import { RedisModule } from '@liaoliaots/nestjs-redis';
+import { RedisModule } from '@songkeys/nestjs-redis';
 import { BullBoardModule } from '@bull-board/nestjs';
 import { BullMQAdapter } from '@bull-board/api/bullMQAdapter';
 import { ExpressAdapter } from '@bull-board/express';
 import { MulterModule } from '@nestjs/platform-express';
 import { ApiController } from './api.controller';
 import { DevelopmentController } from './development.controller';
-import { QueueConstants } from '../../../libs/common/src';
+import { ASSET_QUEUE_NAME, BATCH_QUEUE_NAME, BROADCAST_QUEUE_NAME, PROFILE_QUEUE_NAME, PUBLISH_QUEUE_NAME, REACTION_QUEUE_NAME, REPLY_QUEUE_NAME, REQUEST_QUEUE_NAME, STATUS_QUEUE_NAME, TOMBSTONE_QUEUE_NAME, TRANSACTION_RECEIPT_QUEUE_NAME, UPDATE_QUEUE_NAME } from '../../../libs/common/src';
```

Comment:
linter errored about not allowing namespaces anymore, so most of the changes are related to removal of namespace and importing types directly

Reply:
If you want this change to be smaller you can do this instead on these:

```suggestion
import * as QueueConstants from '../../../libs/common/src';
```